torchci/lib/bot/utils.ts

```diff
@@ -17,7 +17,8 @@ export function repoKey(context: Context): string {
 }
 
 export function isPyTorchManagedOrg(owner: string): boolean {
```

Comment:
Perhaps this function name should be changed to something like `isMergebotSupportedOrg`?